public/static/robots.txt

```diff
@@ -1,105 +1,16 @@
-# ### Fetched from ndla.no 09.09.2018 with additions ###
-#
-# $Id: robots.txt,v 1.9.2.1 2008/12/10 20:12:19 goba Exp $
-#
-# robots.txt
-#
-# This file is to prevent the crawling and indexing of certain parts
-# of your site by web crawlers and spiders run by sites like Yahoo!
-# and Google. By telling these "robots" where not to go on your site,
-# you save bandwidth and server resources.
-#
-# This file will be ignored unless it is at the root of your host:
-# Used:    http://example.com/robots.txt
-# Ignored: http://example.com/site/robots.txt
-#
-# For more information about the robots.txt standard, see:
-# http://www.robotstxt.org/wc/robots.html
-#
-# For syntax checking, see:
-# http://www.sxw.org.uk/computing/robots/check.html
-
+# ### Updated ndla.no 03.04.2024 with additions ###
 User-agent: *
-Crawl-delay: 10
-# Directories
-Disallow: /includes/
-Disallow: /misc/
-Disallow: /modules/
-Disallow: /profiles/
-Disallow: /scripts/
-Disallow: /sites/
-Disallow: /themes/
-# Files
-Disallow: /CHANGELOG.txt
-Disallow: /cron.php
-Disallow: /INSTALL.mysql.txt
-Disallow: /INSTALL.pgsql.txt
-Disallow: /install.php
-Disallow: /INSTALL.txt
-Disallow: /LICENSE.txt
-Disallow: /MAINTAINERS.txt
-Disallow: /update.php
-Disallow: /UPGRADE.txt
-Disallow: /xmlrpc.php
-# Paths (clean URLs)
-Disallow: /admin/
-Disallow: /comment/reply/
-Disallow: /contact/
-Disallow: /logout/
-Disallow: /node/add/
-Disallow: /search/
-Disallow: /user/register/
-Disallow: /user/password/
-Disallow: /user/login/
-# Paths (no clean URLs)
-Disallow: /?q=admin/
-Disallow: /?q=comment/reply/
-Disallow: /?q=contact/
-Disallow: /?q=logout/
-Disallow: /?q=node/add/
-Disallow: /?q=search/
-Disallow: /?q=user/password/
-Disallow: /?q=user/register/
-Disallow: /?q=user/login/
-# Added by CERPUS
-Disallow: /print/
-Disallow: /printpdf/
-Disallow: /?q=print/
-Disallow: /?q=printpdf/
-Disallow: /tracker/
-Disallow: /?q=tracker/
-
-
-
-Disallow: /*/ndla_utils/menu2010/grep/
-Disallow: /*/*/*/grep/uuid:*
-Disallow: /ndla_ontopia_connect_show_aim/
-
-# Allow to see 404/301/302 from old structure
-Allow: /*/easyreader/
-Allow: /*/biblio
-Allow: /*/lightbox
-Allow: /*/contentbrowser
```

Comment:
Vi håndterer fremdeles disse der vi sender 301 eller 404 avhengig av om vi finner noko i taksonomi.

# Allow to see 404/301/302 from old structure
Allow: /*/node/
Allow: /*/printpdf/
Allow: /*/print/
Allow: /*/easyreader/
Allow: /*/biblio/
Allow: /*/lightbox/
Allow: /*/contentbrowser/
Allow: /*/aktualitet/
Allow: /*/oppgave/
Allow: /*/fagstoff/

Reply:
Men dei er vel opne som default?

Reply:
Dei er kanskje det? Eg aner egentlig ikkje korleis dette fungerer.